plugins/reporters/web-app/src/funTest/kotlin/WebAppReporterFunTest.kt

```diff
@@ -36,7 +36,7 @@ class WebAppReporterFunTest : WordSpec({
 
             val report = WebAppReporter().generateReport(ReporterInput(ortResult), outputDir).single()
 
-            report.length() should beInRange(2140000L..2160000L)
+            report.length() should beInRange(1870000L..1880000L)
```

Comment:
Any idea why the report became so much smaller? This looks a bit suspicious, to be honest.